contollers/bookings.ts

```diff
@@ -0,0 +1,90 @@
+import { Request, Response } from "express";
+import { bookingType } from "../utils/types";
+import validator from "email-validator";
+import moment from "moment";
+
+//Use this array as your (in-memory) data store.
+let bookings: bookingType[] = require("../bookings.json");
+
+const validateBook = (booking: bookingType) => {
+  if (
+    !booking.title ||
+    !booking.firstName ||
+    !booking.surname ||
+    !booking.email ||
+    !validator.validate(booking.email) ||
+    !booking.roomId ||
+    !booking.checkInDate ||
+    !booking.checkOutDate ||
+    !moment(booking.checkOutDate).isAfter(booking.checkInDate)
+  ) {
+    return false;
+  }
+  return true;
```

Comment:
would be more clean if we use object destructuring:

```
const { title, firstName, surname, email, roomId, checkInDate, checkOutDate } = booking;
if ( 
    !title || 
    !firstName || 
    !surname || 
    !email || 
    !emailValidator.validate(booking.email) ||
    !roomId || 
    !checkInDate || 
    !checkOutDate ||
    !moment(checkOutDate).isAfter(checkInDate)
) {
    return false;
}
```

Reply:
I see. It's because if I use your coding style with Prettier, it will result in even more lines of code.

Reply:
number of lines is not a problem, the problem is repeating the code (booking.xxx)